src/common/components/repository-row/dropdowns/template-yaml.js

```diff
@@ -1,18 +1,33 @@
-export default `
-name: my-snap-name # you probably want to 'snapcraft register <name>'
-version: '0.1' # just for humans, typically '1.2+git' or '1.3.2'
-summary: Single-line elevator pitch for your amazing snap # 79 char long summary
-description: |
-  This is my-snap's description. You have a paragraph or two to tell the
-  most important story about your snap. Keep it under 100 words though,
-  we live in tweetspace and your description wants to look good in the snap
-  store.
+export default function templateYaml(name, storeName) {
+  let repoName = name;
```

Comment:
This assignment isn't needed. You could either set `templateYaml(repoName,...` if you wanted a better var than `name` or just use `name` throughout the code.